CHANGELOG.md

```diff
@@ -122,16 +122,24 @@ the APIs in these libraries are stable, and are ready for production use.
 
 New `*Client` for conversational search.
 
-### [Policy Troubleshooter](/google/cloud/policytroubleshooter/README.md)
+### [OpenTelemetry](/google/cloud/opentelemetry/README.md)
 
-Remove `PoliciesClient`. This client was placed in the wrong library. The
-correct location is google/cloud/iam/v2.
+Features to instrument the Google Cloud C++ libraries for [distributed tracing]
+using [OpenTelemetry] are now GA. See the [quickstart][otel-quickstart] within
+the library to learn more about these tools.
 
 ### Known issues
 
 Compiling gRPC with Bazel and Clang >= 16 requires `--features=-layering_check`
 in your Bazel command-line. For more details, see [grpc#34482].
 
+## v2.15.1 - 2023-09
```

Comment:
I don't think we try to document the branches from the main line.  The HEAD-ancestor that did this work (#12585) was not from v2.15.1.

Reply:
Ah, I see. Thanks.